### PR TITLE
Add more documentation to datastore_wordcount example

### PR DESCRIPTION
**Pull request by vikkyrk**
Be sure to do all of the following to help us incorporate your contribution
quickly and easily:

 - [ ] Make sure the PR title is formatted like:
   `[BEAM-<Jira issue #>] Description of pull request`
 - [ ] Make sure tests pass via `mvn clean verify`. (Even better, enable
       Travis-CI on your fork and ensure the whole test matrix passes).
 - [ ] Replace `<Jira issue #>` in the title with the actual Jira issue
       number, if there is one.
 - [ ] If this contribution is large, please file an Apache
       [Individual Contributor License Agreement](https://www.apache.org/licenses/icla.txt).

---

### CODE REVIEW

**Comment by vikkyrk** [comment]
R: @aaltay

File: sdks/python/apache_beam/examples/cookbook/datastore_wordcount.py [hunk 1]
```diff
@@ -15,7 +15,49 @@
 # limitations under the License.
 #
 
-"""A word-counting workflow that uses Google Cloud Datastore."""
+"""A word-counting workflow that uses Google Cloud Datastore.
+
+This example shows how to use ``datastoreio`` to read from and write to
+Google Cloud Datastore. Note that running this example may incur charge for
```
Comment (aaltay) [inline sdks/python/apache_beam/examples/cookbook/datastore_wordcount.py]: not sure, but is it supposed to be `incur charges`?

File: sdks/python/apache_beam/examples/cookbook/datastore_wordcount.py [hunk 2]
```diff
@@ -15,7 +15,49 @@
...
+Cloud Datastore operations.
+
+See https://developers.google.com/datastore/ for more details on Google Cloud
+Datastore.
+See https://cloud.google.com/dataflow/docs/quickstarts/quickstart-python on
```
Comment (aaltay) [inline sdks/python/apache_beam/examples/cookbook/datastore_wordcount.py]: Should this be `http://beam.incubator.apache.org/get-started/quickstart/` instead?

Comment (vikkyrk) [inline sdks/python/apache_beam/examples/cookbook/datastore_wordcount.py]: Beam doesn't have instructions on how to run on Dataflow Service.

File: sdks/python/apache_beam/examples/cookbook/datastore_wordcount.py (hunk 1 above)
Comment (vikkyrk) [inline sdks/python/apache_beam/examples/cookbook/datastore_wordcount.py]: Yes, if you exceed the daily quota of read and writes. https://cloud.google.com/appengine/docs/quotas#Datastore

File: sdks/python/apache_beam/examples/cookbook/datastore_wordcount.py (hunk 2 above)
Comment (vikkyrk) [inline sdks/python/apache_beam/examples/cookbook/datastore_wordcount.py]: Fixed

**Comment by aaltay** [comment]
LGTM

**Comment by vikkyrk** [comment]
R: @robertwb

**Pull request closed without merging**